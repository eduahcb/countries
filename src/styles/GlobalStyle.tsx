import { createGlobalStyle } from 'styled-components'

const Style = createGlobalStyle`

  :root {
    --text-primary: #111517;
    --input: hsl(0, 0%, 52%);
    --background: #FAFAFA;
    --elements: #FFFFFF;
  }

  html {
    font-family: 'Nunito Sans', sans-serif;
    box-sizing: border-box;
    font-size: 1rem;
    font-weight: 300;
  }

  *, *::before, *::after {
    margin: 0;
    padding: 0;
    box-sizing: inherit;
  }

  body {
    background: var(--background);
    height: 100vh;
  }

  a, input, button, textarea {
    font-family: 'Poppins', sans-serif;
  }

  h1, h2, h3, h4, h5, h6, strong {
    font-weight: 800;
  }

`

export const GlobalStyle = Style
