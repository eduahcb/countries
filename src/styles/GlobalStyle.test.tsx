import React from 'react'
import { render } from '@testing-library/react'

import { GlobalStyle } from './GlobalStyle'

describe('GlobalStyle', () => {
  test('match snapshot', () => {
    const { asFragment } = render(<GlobalStyle />)

    expect(asFragment()).toMatchSnapshot()
  })
})
