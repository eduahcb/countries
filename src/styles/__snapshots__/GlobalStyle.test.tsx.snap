// Vitest Snapshot v1

exports[`GlobalStyle > match snapshot 1`] = `<DocumentFragment />`;
